Automattic/post-queue
# Fix quick edit, bulk edit, and post list reordering

Fixes https://github.com/Automattic/wp-post-queue/issues/1

This PR makes the following changes:

* Adds the 'Queued' status to the Quick Edit status drop down
* Adds the 'Queued' status to the Bulk Edit drop down
* Fixes drag-and-drop reorder when there are enough posts in the queue to have pagination

## Testing Instructions
* Install the build of this branch ([wp-post-queue.zip](https://github.com/user-attachments/files/17561127/wp-post-queue.zip))
* Make sure you have a few drafts saved
* Go to the drafts post list page and quick edit one of the posts
* See that you can update its state to 'Queued'
* Now do the same using the Bulk Editor
* Go to the queued page and make sure you can drag and drop as expected
* Test using Quick Edit to change post details
* Test using Quick Edit to change post status back to draft
* Test using Bulk Edit to change post status back to draft
